cvprac/cvp_client.py

```diff
@@ -170,7 +171,7 @@ def set_log_level(self, log_level='INFO'):
         self.log.setLevel(getattr(logging, log_level))
 
     def connect(self, nodes, username, password, connect_timeout=10,
-                protocol='http', port=None):
+                port=None, cert=False):
```

Comment:
I could change the default for cert to None. None or path to file would make a bit more sense than bool or path to file. I would just have to test this to verify I can pass None as the verify parameter to the requests get method.